SE-XXX Typed throws.md

```diff
@@ -412,13 +412,42 @@ Everything that applies to the error type of `Result` also applies to error type
 ### Swift grammar additions/changes
 
 What this proposal is about is giving resilience and type safety to an area of the Swift language that lacks of it, ganing both in safety for the developer not just in type system but in terms of reducing the number of possible path error recovering that the developer might have to face when consuming an API.
-The proposed semantics are pretty simple and additive from which we have today:
+
+We define the proposed semantics represented as how the function would be mangled or just as a pseudocode:
 
 ```
 function-signature ::= params-type params-type throws? throws-type?
 ```
 
-In the snippet below, we try to ilustrate which would be the end result of the implementation of the current proposal:
+This gets translated into a more Swifty grammar that would be:
+
+```swift
+func/init(params: params-type...) (throws throws-type?)? (-> return-type)
+```
+
+Where, following our kitty example, we could write:
+
+```swift
+// From:
+/// throws CatError
+func callCatOrThrow() throws -> Cat
+
+// To:
+func callCatOrThrow() throws CatError -> Cat
+
+// Being CatError:
+struct CatError: Swift.Error {}
+```
+
+### The grammar rules
+
+In the example above, despite being pretty simple, stablishes all the rules that applies to typed throwing functions:

```

Comment:
stablishes?

that apply?

Reply:
How would you write this sentence. "Given those rules, all the following content will follow those rules"